### PR TITLE
add claim page

### PR DESCRIPTION
**Pull request by cctdaniel**
choose "Use browser wallet" instead of "metamask" because if you have phantom installed, apparently phantom overrides metamask so it shows metamask not found, the connectkit team is aware of this but no further info on a fix

### CODE REVIEW

File: frontend/pages/staking.tsx [hunk 1]
```diff
@@ -1,36 +1,36 @@
...
   VestingAccountState,
-} from "pyth-staking-api";
-import { ChangeEvent, Fragment, useEffect, useState } from "react";
-import toast from "react-hot-toast";
-import { capitalizeFirstLetter } from "utils/capitalizeFirstLetter";
-import { classNames } from "utils/classNames";
-import Layout from "../components/Layout";
-import SEO from "../components/SEO";
-import { getPythTokenBalance } from "./api/getPythTokenBalance";
+} from 'pyth-staking-api'
```
Comment (guibescos) [inline frontend/pages/staking.tsx:21]: This new formatting is unfortunate, what triggered it?

Comment (guibescos) [reply frontend/pages/staking.tsx:21]: This makes it hard to review

**Review by guibescos — COMMENTED** [review]
The sign in button doesn't work right, because there's no server?

**Comment by cctdaniel** [comment]
> The sign in button doesn't work right, because there's no server?

does it not work for you? it should work

File: frontend/pages/_app.tsx [hunk 2]
```diff
@@ -32,6 +53,12 @@ const App: FC<AppProps> = ({ Component, pageProps }: AppProps) => {
   // You can also provide a custom RPC endpoint
   // const endpoint = useMemo(() => clusterApiUrl(network), [network])
 
+  const signerOptions: SignerOptions = {
```
Comment (guibescos) [inline frontend/pages/_app.tsx:56]: Delete this?

**Review by guibescos — DISMISSED** [review]

**Review by guibescos — APPROVED** [review]

**Pull request merged**